types/eth-url-parser/tslint.json

```
@@ -0,0 +1,14 @@
+{ 
+  "extends": "@definitelytyped/dtslint/dt.json",
+  "rules": {
+    "npm-naming": [
+      true,
+      {
+        "mode": "code",
+        "errors": [
+          ["NeedsExportEquals", false]
+        ]
+      }
+    ]
+  }
```

```suggestion
  "extends": "@definitelytyped/dtslint/dt.json"
```
why do you need those exclusions? Your code looks OK and should work without those

---

Thanks for the review @peterblazejewicz! This is a current issue with dtslint's `NeedExportEquals` rule, see https://github.com/DefinitelyTyped/DefinitelyTyped/issues/62444 and the matching Discord thread https://discord.com/channels/508357248330760243/757992230077333655/1023217663608094720.

Confirmed by @sandersn on Discord the types are sane and this is a false positive. Just gave it another try reverting the rule ignore and unfortunately, the issue is still happening on latest master so we still need to disable the rule.